### ISSUE TITLE
v2023.9.0: Blank screen 

### ISSUE THREAD

**Issue by mindo** [jitsi/jitsi-meet-electron#908]
### Description:

After the upgrade to [2023.9.0](https://github.com/jitsi/jitsi-meet-electron/releases/tag/v2023.9.0) I get a blank screen as soon as I click anywhere inside the jitsi window.

### Steps to reproduce:

1. Open jitsi
2. Click anywhere / wait a few seconds
3. Blank screen

### Expected behavior:

I should be able to connect to any server / room

### Actual behavior:

I get a blank screen.

### Server information:

This happens even before entering a url/room.

### Client information:

- Browser / app version: Appimage/deb 2023.9.0
- Operating System: Fedora 38 (wayland) + Kali Linux (x11)
![fedora_wayland_appimage](https://github.com/jitsi/jitsi-meet/assets/1084642/5f1259d8-3b5f-42a2-a62f-cd7b2f5e3a42)
![kali_x11_appimage](https://github.com/jitsi/jitsi-meet/assets/1084642/8c3ff3da-9bb3-40cd-ac55-32267582adeb)
![kali_x11_deb](https://github.com/jitsi/jitsi-meet/assets/1084642/ef05f660-bbfe-431e-9647-056dc39cbdec)

### Additional information:

Fedora logs:

`
➜ ./jitsi-meet-x86_64_bad.AppImage --enable-logging
Gtk-Message: 10:23:43.187: Failed to load module "appmenu-gtk-module"
10:23:43.356 › Checking for update
[1594135:0907/102343.518635:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
[1594044:0907/102343.537238:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
[1594044:0907/102344.129743:INFO:CONSOLE(2)] "%c action %cpersist/PERSIST %c@ 10:23:44.128 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.129774:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.129798:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.129808:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.129823:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.132653:INFO:CONSOLE(2)] "%c action %cpersist/REHYDRATE %c@ 10:23:44.132 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.132679:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.132705:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.132720:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.132735:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.138481:INFO:CONSOLE(2)] "%c action %c@@router/LOCATION_CHANGE %c@ 10:23:44.138 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.138507:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.138525:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.138535:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.138548:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.238390:INFO:CONSOLE(2)] "%c action %cSymbol(START_ONBOARDING) %c@ 10:23:44.235 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.238440:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.238453:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.238467:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
[1594044:0907/102344.238477:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-a7mqej/resources/app.asar/build/app.js (2)
10:23:44.431 › Update for version 2023.9.0 is not available (latest version: 2023.9.0, downgrade is disallowed).
10:23:44.431 › checkForUpdatesAndNotify called, downloadPromise is null
Renderer process crashed - see https://www.electronjs.org/docs/tutorial/application-debugging for potential debugging information.
[1594135:0907/102354.257944:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[1594135:0907/102414.914816:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[1594135:0907/102507.622692:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
`

Kali logs:

`
─$ ./jitsi-meet-x86_64.AppImage --enable-logging
05:22:20.958 › Checking for update
[118372:0907/052221.137869:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
[118417:0907/052221.166624:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
[118417:0907/052221.252167:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[118417:0907/052221.269713:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[118372:0907/052221.746433:INFO:CONSOLE(2)] "%c action %cpersist/PERSIST %c@ 05:22:21.745 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.746467:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.746500:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.746529:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.746553:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.749723:INFO:CONSOLE(2)] "%c action %cpersist/REHYDRATE %c@ 05:22:21.749 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.749750:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.749775:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.749792:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.749812:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.755700:INFO:CONSOLE(2)] "%c action %c@@router/LOCATION_CHANGE %c@ 05:22:21.755 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.755730:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.755760:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.755778:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.755798:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.805000:INFO:CONSOLE(2)] "%c action %cSymbol(START_ONBOARDING) %c@ 05:22:21.804 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.805056:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.805079:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.805099:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118372:0907/052221.805112:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-oEc5Qe/resources/app.asar/build/app.js (2)
[118417:0907/052221.880921:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
05:22:23.307 › Update for version 2023.9.0 is not available (latest version: 2023.9.0, downgrade is disallowed).
05:22:23.308 › checkForUpdatesAndNotify called, downloadPromise is null
Renderer process crashed - see https://www.electronjs.org/docs/tutorial/application-debugging for potential debugging information.
`

**Comment by saghul** [comment]
![Screenshot_20230908_090703](https://github.com/jitsi/jitsi-meet-electron/assets/317464/e653f124-d390-4001-8a77-906f10e81299)

I am unable to reproduce this on Arch :-/

**Comment by q-b** [comment]
Confirmed, I use jitsi from flathub on Ubuntu 20.04, same behavior (blank screen after a few secondes, no click needed)

**Comment by saghul** [comment]
Please test with our builds to avoid confusion, the Flatpak build is a 3rd party build.

**Comment by q-b** [comment]
Same with AppImage from here

```
❯ ./jitsi-meet-x86_64.AppImage        
11:17:44.401 › Checking for update
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
11:17:44.600 › Generated new staging user ID: d834ad73-b03b-58e6-b9db-bf3c3d0dc528
11:17:45.943 › Update for version 2023.9.0 is not available (latest version: 2023.9.0, downgrade is disallowed).
11:17:45.944 › checkForUpdatesAndNotify called, downloadPromise is null
[14604:0908/111748.902077:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[14604:0908/111752.161131:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
```

Firefox is working fine though...

**Comment by saghul** [comment]
Can you try and get the main window console logs?

**Comment by q-b** [comment]
Found this issue on electron : https://github.com/electron/electron/issues/32760

The FreeTube/NixOS issue seems to have the same behaviour (white screen)

**Comment by q-b** [comment]
```
❯ ELECTRON_ENABLE_LOGGING=true ./jitsi-meet-x86_64.AppImage 
12:08:19.435 › Checking for update
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[19366:0908/120819.581266:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
[19324:0908/120819.661768:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
[19324:0908/120820.210267:INFO:CONSOLE(2)] "%c action %cpersist/PERSIST %c@ 12:08:20.208 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.210302:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.210325:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.210339:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.210356:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.213017:INFO:CONSOLE(2)] "%c action %cpersist/REHYDRATE %c@ 12:08:20.212 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.213029:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.213041:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.213049:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.213069:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.227234:INFO:CONSOLE(2)] "%c action %c@@router/LOCATION_CHANGE %c@ 12:08:20.219 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.227280:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.227299:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.227319:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.227334:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265180:INFO:CONSOLE(2)] "%c action %cSymbol(START_ONBOARDING) %c@ 12:08:20.263 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265233:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265249:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265269:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265281:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265295:INFO:CONSOLE(2)] "%c action %cSymbol(SET_ACTIVE_ONBOARDING) %c@ 12:08:20.264 color: gray; font-weight: lighter; color: inherit; color: gray; font-weight: lighter;", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265307:INFO:CONSOLE(2)] "%c prev state color: #9E9E9E; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265322:INFO:CONSOLE(2)] "%c action     color: #03A9F4; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265333:INFO:CONSOLE(2)] "%c next state color: #4CAF50; font-weight: bold [object Object]", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
[19324:0908/120820.265347:INFO:CONSOLE(2)] "console.groupEnd", source: file:///tmp/.mount_jitsi-ITPkup/resources/app.asar/build/app.js (2)
12:08:20.801 › Update for version 2023.9.0 is not available (latest version: 2023.9.0, downgrade is disallowed).
12:08:20.802 › checkForUpdatesAndNotify called, downloadPromise is null
Renderer process crashed - see https://www.electronjs.org/docs/tutorial/application-debugging for potential debugging information.
[19366:0908/120823.875895:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
```

**Comment by csett86** [comment]
Hmmm, really strange, I see it working fine with Ubuntu 22.04 (Wayland)

![grafik](https://github.com/jitsi/jitsi-meet-electron/assets/1392875/ee4ccd5d-d2a8-4515-8155-6e09aa14e76e)

**Comment by dutty5** [comment]
Same issue here (white screen in a few seconds after the start, no click needed) on Fedora 38 (wayland), both with Flatpak version and AppImage downloaded from here (version 2023.9.0). Any solution? Thanks.

**Comment by csett86** [comment]
This is very strange, I tried to reproduce why the renderer crashes for all of you, but no luck (Fedora 38, Wayland, in VirtualBox). Both AppImage and flatpak work fine. Maybe something with the graphics driver (as its software rendering inside VirtualBox...)

![Bildschirmfoto 2023-09-09 um 22 00 23](https://github.com/jitsi/jitsi-meet-electron/assets/1392875/a5b53937-f2d5-44b0-a46f-6f70980f9bf1)

**Comment by dutty5** [comment]
> This is very strange, I tried to reproduce why the renderer crashes for all of you, but no luck (Fedora 38, Wayland, in VirtualBox). Both AppImage and flatpak work fine. Maybe something with the graphics driver (as its software rendering inside VirtualBox...)

I'm not sure if ever something crashes at all. In my previous post on the screenshot the Jitsi-meet screen is already blank but no crash or error reported in the command line in the background Terminal window.

I run pretty stock Fedora 38 Gnome on physical machine Lenovo X1 Nano with Intel graphics. No nvidia drivers involved.

**Comment by q-b** [comment]
Intel iGPU here aswell (I can check the driver used if required)

**Comment by cedricroijakkers** [comment]
Chiming in here, I can reproduce the issue too. I've tried installing the binary builds, or building Jitsi myself.

Environment:
Arch Linux, GNOME, X11, Intel GPU

**Comment by adolfintel** [comment]
Same issue here on Arch, but building with electron25 instead of 26 fixes it.

**Comment by cedricroijakkers** [comment]
> Same issue here on Arch, but building with electron25 instead of 26 fixes it.

Can confirm that this works. The following patch to the `PKGBUILD` file makes the application work again as a quick workaround:

```patch
diff --git a/PKGBUILD b/PKGBUILD
index d8edf5e..a27d8f4 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -13,11 +13,11 @@ license=('Apache')
 conflicts=('jitsi-meet-electron-bin'
            'jitsi-meet-electron')
 replaces=('jitsi-meet-electron')
-_electron_pkg="electron26"
+_electron_pkg="electron25"
 depends=('gtk3'
          'libxss'
          'nss'
-         "${_electron_pkg}>=26")
+         "${_electron_pkg}>=25")
 makedepends=('coreutils'
              'git'
              'python'
```

**Comment by arjanoosting** [comment]
Ferdium had the [same issue](issue) and [rolled back to 25](https://github.com/ferdium/ferdium-app/pull/1341)

**Comment by saghul** [comment]
Does installing the "vainfo" package solve it for those affected?

**Comment by adolfintel** [comment]
@saghul nope

**Comment by saghul** [comment]
@csett86 Since 25 is still supported how about a downgrade?

**Comment by csett86** [comment]
We can roll-back to electron 25, then we loose the wayland screensharing fixes of electron 26 (https://github.com/electron/electron/pull/39111, as the backport of that is not yet merged in electron 25: https://github.com/electron/electron/pull/39710), but regain basic functionality with Fedora and Arch, I guess.

The renderer crash is tracked upstream as https://github.com/electron/electron/issues/39775, so I will revert and only upgrade to electron 26 once the renderer crash is fixed

**Comment by saghul** [comment]
SGTM!

**Comment by csett86** [comment]
FYI: Released v2023.9.1 with the downgrade just now
